docs/SOP.md

```diff
@@ -6,14 +6,15 @@ title: Standard Operating Procedures
 
 This document contains standard operating procedures (SOPs) for the OBO Foundry Operations Committee, and is intended for internal use only.
 
-## SOPs
+# SOPs
```

Comment:
We just made this change the other way around, because `#` (headline 1) is  reserved to the "site title" of a page. So we should keep this as it was.